src/Filters/LinearKalmanFilter.cpp

```diff
@@ -0,0 +1,87 @@
+#include "LinearKalmanFilter.h"
+
+namespace mmfs
+{
+
+    // Constructor initializes sizes
+    LinearKalmanFilter::LinearKalmanFilter(Matrix X, Matrix U, Matrix P, Matrix F, Matrix G, Matrix R, Matrix Q)
```

Comment:
We need to check the size of these matrices. they are all defined based on, size of state, size of control and number of measurements. We are missing the information for the number of measurements. We don't set an H here.

Reply:
I could overload the constructor and have it take in the sizes and it initializes them to a random (maybe empty or 0s) matrices, if that would be helpful. If they're the wrong size and passed in on this constructor though, what should be the behavior, we just use the error logging and throw an error?